### PR TITLE
Add slotscheck

### PR DESCRIPTION
**Pull request by q0w**
What about the configuration?
# Checklist

- [ ] Changes finished
- [ ] Tests added (if applicable)
- [ ] Documentation adjusted (if applicable)
- [ ] Added entry to changelog (`HISTORY.rst`)
- [ ] `tox` runs OK
- [ ] Build runs OK

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -23,36 +23,43 @@ commands=
...
 
 [testenv:isort]
 basepython=python3.9
 commands=
-  poetry run isort --check-only --diff src/ tests/
+    poetry run isort --check-only --diff src/ tests/
+
+[testenv:slotscheck]
+basepython=python3.9
+commands_pre=
```
Comment (ariebovenberg) [inline tox.ini:36]: defining `commands_pre` should not be needed. Isn't it inherited from the base testenv?

**Review by ariebovenberg — CHANGES_REQUESTED** [review]
Nice! Just one more question 🤔

File: tox.ini (hunk 1 above)
Comment (q0w) [reply tox.ini:36]: So base **commands_pre** does not install project itself, but its required for slotscheck to properly import snug module https://github.com/ariebovenberg/snug/pull/367#issuecomment-1016992276

Comment (ariebovenberg) [reply tox.ini]: correct, `poetry` does not install the project, but `tox` runs `pip install snug` internally 🤔.

Removing it worked "on my machine", so let's see what GH actions make of it

**Review by ariebovenberg — APPROVED** [review]

**Pull request merged**